src/lib/stores.ts

```diff
@@ -31,9 +31,9 @@ ColorSpace.register(REC_2020);
 export { ColorSpace };
 
 export const INITIAL_VALUES = {
-  format: 'oklch' as ColorFormatId,
-  bg_coord: [0.3259, 0.1333, 265.49] as [number, number, number],
-  fg_coord: [0.8241, 0.1061, 0.8587] as [number, number, number],
+  format: 'p3' as ColorFormatId,
+  bg_coord: [0.0967, 0.167, 0.4494] as [number, number, number],
+  fg_coord: [0.951, 0.675, 0.7569] as [number, number, number],
```

Comment:
@jamesnw @mirisuzanne What do you think about this change so that we default to P3 (for now)?

Reply:
I'm going back and forth on this- I think we want to be careful in what we're communicating. The Oklch space itself is not the problem- the gamut mapping algorithm is broken in a way that can impact p3 as well, but because of the wideness of the ok gamut and the promises about perceptual continuity, it is much more apparent with Oklch and Oklab. So I don't want to communicate "don't use Oklch, it's broken". But slowing adoption of Oklch until gamut mapping is fixed could be useful, and so temporarily switching the default could make sense. In other words, 🤷‍♂️ .

Reply:
Agree with all of that. I think the switch is fine for now.